<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="16" skipped="0" tests="74" time="843.167" timestamp="2025-08-30T12:53:39.051726" hostname="dut7358"><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorMetadata" name="test_serialize_and_deserialize" time="0.122" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestCreateTensorFromParams" name="test_empty" time="0.002" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardParameter" name="test_shard_parameter" time="3.811"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2434507">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2434507</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardParameter" name="test_shard_parameter_errors" time="3.809"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2434812">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2434812</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardTensor" name="test_shard_tensor" time="3.810"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2435116">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2435116</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardTensor" name="test_shard_tensor_errors" time="3.710"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2435420">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2435420</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardTensor" name="test_shard_tensor_with_empty_shard" time="3.810"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2435725">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2435725</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestModuleHookApi" name="test_collect_local_shard" time="16.131" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestModuleHookApi" name="test_reshard_output" time="3.710"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2436330">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2436330</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestLocalTensor" name="test_local_tensor" time="16.230" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestLocalTensor" name="test_local_tensor_error" time="16.231" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_cleanup" time="16.630" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_complete_world_size" time="16.631" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_create_sharded_tensor_like" time="16.731" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_create_sharded_tensor_with_full" time="16.231" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_create_sharded_tensor_with_ones" time="16.632" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_create_sharded_tensor_with_rand" time="16.631" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_create_sharded_tensor_with_zeros" time="16.633" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_gather_even" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_gather_uneven" time="16.432" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_insufficient_sharding_dims" time="3.310" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_invalid_pg_rpc_ranks" time="3.710" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_invalid_sharding" time="3.510" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_load_state_dict_errors" time="3.811" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_multiple_local_shards" time="16.632" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_new_group" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_partial_world_size" time="16.431" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_sharded_tensor_metadata" time="16.631" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_sharded_tensor_sizes" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_sharding_columns" time="3.610" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_state_dict" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_state_dict_new_group" time="28.849" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorChunked" name="test_state_dict_no_sharded_tensors" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_create_sharded_tensor_with_ones" time="16.525" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_gather_even" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_gather_uneven" time="16.732" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_grid_sharding" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_multiple_local_shards" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_new_group" time="16.330" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_partial_world_size" time="16.431" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_sharded_tensor_device" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_sharded_tensor_metadata" time="16.531" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_sharded_tensor_to_accelerator" time="16.833" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_sharded_tensor_to_cpu" time="16.431" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_sharded_tensor_to_test" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_uneven_shards" time="3.310" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorEnumerable" name="test_with_rpc_names" time="16.330" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalTensor" name="test_init_from_local_tensor" time="16.731" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalTensor" name="test_init_from_local_tensor_errors" time="16.231" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards" time="4.211"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2454716">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2454716</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_and_global_metadata" time="16.731" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_and_global_metadata_invalid_shards" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_and_global_metadata_with_all_zeros" time="15.930" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_and_global_metadata_with_local_view" time="16.130" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_invalid_local_shards" time="4.111"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2456744">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2456744</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_invalid_pin_memory" time="3.710"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2457232">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2457232</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_invalid_property_cross_ranks" time="4.110"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2457536">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2457536</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_invalid_shards_gaps" time="4.111"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2458012">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2458012</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_invalid_shards_overlap" time="4.312"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2458488">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2458488</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_new_group" time="17.131" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_init_from_local_shards_with_different_glb_size" time="3.811"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2459442">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2459442</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_local_shards" time="16.031" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_non_rw_sharded_recalc_for_metadata" time="3.811"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2460047">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2460047</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_recalc_for_metadata" time="3.912"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2460351">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2460351</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorFromLocalShards" name="test_st_base_init_from_local_shards_and_global_metadata" time="3.410" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorCustomOps" name="test_custom_op" time="16.632" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorCustomOps" name="test_custom_op_errors" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorCustomOps" name="test_custom_op_override" time="4.111"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2461885">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2461885</failure></testcase><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardMetadata" name="test_create_shard_with_no_placement" time="16.431" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardMetadata" name="test_shard_metadata_init" time="16.331" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorSubGroupInit" name="test_sub_process_group_placement_validation" time="0.027" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestShardedTensorSubGroupInit" name="test_sub_process_group_sharded_tensor_init" time="0.007" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestCreateTensorNoProcessGroupMode" name="test_init_from_local_shards_and_global_metadata" time="0.001" /><testcase classname="test.distributed._shard.sharded_tensor.test_sharded_tensor.TestCreateTensorNoProcessGroupMode" name="test_non_contiguous_local_shards" time="0.001" /></testsuite></testsuites>